ja.po: ch_arch_storage.xml

amotoki opened the pull request.

amotoki committed:
ja.po: ch_arch_storage.xml

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:34
+msgid "Ephemeral storage"
+msgstr "エフェメラル・ストレージ"
```
s/エフェメラル・ストレージ/エフェメラルストレージ

https://github.com/openstack-ja/openstack-manuals/wiki/TranslationGuideline
によると、

> 二語以上からなるキーワードは原則「中黒（・）なし」を使う。商標等で必要があれば付けることも可能

とのことです。

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:42
+msgid "Run operating system and scratch space"
+msgstr "OSを起動するため"
```
「and scratch space」(スクラッチ領域)の訳は不要でしょうか。
(いい日本語訳が思い浮かびません。。)

yosshy replied:
→OS を起動し、空き領域に記録する。

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:88
+msgid "Example of typical usage…"
+msgstr "典型的な利用方法"
```
s/利用方法/利用例
かと思います。

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"underlying storage hardware's block device. This helps increase the overall "
+"read/write IO."
+msgstr ""
+"多くのストレージ・ドライバはインスタンスが直接ストレージハードウェアへアクセ"
```
s/ハードウェアへ/ハードウェアのブロックデバイスへ

としたほうが正確かなと思いました。

yosshy replied:
"underlying" が抜けています。
→ほとんどのブロックストレージドライバは、インスタンスが下位層のストレージハードウェアのブロックデバイスにアクセスできるようにします。

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"Cinder. By Grizzly's release, this has expanded into a full NFS driver as "
+"well as a GlusterFS driver."
+msgstr ""
+"Folsomでは実験的なサポートとしてファイルをボリュームとして利用できます。初期"
```
「として」が二回続くので、「実験的なサポートですが、」とするのはどうでしょうか。

yosshy replied:
→Folsom リリースでは、ファイルをボリュームとして使用する為の実験的なサポートを開始しました。

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"translation is similar to how OpenStack utilizes QEMU's file-based virtual "
+"machines stored in <code>/var/lib/nova/instances</code>."
+msgstr ""
+"これらのドライバーは従来のブロック・ストレージドライバとは異なる動作をしま"
```
「少々(もしくは「少し」)異なる動作をします」とa littleを和訳に入れるのはどうでしょうか。

yosshy replied:
s/ブロック・ストレージドライバ/ブロックストレージドライバ/

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+msgid ""
+"With file-level storage, users access stored data using the operating "
+"system's file system interface. Most users, if they have used a network "
+"storage solution before, have encountered this form of networked storage. In "
```
この一文は、翻訳されていないですが、これが無くとも特に意味は通じと思いますが、
若干次のNFS, CIFSの部分が唐突な気もします。

irixjp replied:
すみません、この部分は文脈的にどう訳していいのか分からず、無かったことにしてしまいました＞＜

irixjp replied:
吉山さんからコメントを頂いたので、それを元に直しました。

amotoki replied:
最新版の訳を確認しました。いいですね。

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:307
+msgid "As a scalable, reliable data store for virtual machine images"
+msgstr "スケーラブルで信頼性のある仮想マシンデータストアとして利用する"
```
s/仮想マシン/仮想マシンイメージ
としてはどうでしょうか。

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"create a volume group dedicated to Block Storage volumes. Blocks are created "
+"from LVM logical volumes."
+msgstr ""
+"ブロック・ストレージを収容する各ホストでは、管理者によって事前にブロック・ス"
```
s/管理者によって/管理者は

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"these main traffic flows: <placeholder-1/>"
+msgstr ""
+"オブジェクト・ストレージのネットワークのトラフィックは通常とは異なっているか"
+"もしれません。以下のトラフィックを考慮てください。 <placeholder-1/>"
```
s/考慮てください/考慮してください

masayukig commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"object begins."
+msgstr ""
+"オブジェクト・ストレージはデータを保持するノード間で頻繁に通信を行います。 こ"
+"れは数MB/sのトラフィック で主に他ホストにオブジェクトの存在確認を行いっていま"
```
「小さなクラスタでさえ」ということを入れておいたほうが良いかなと思いました。
数MB/sのトラフィックが、どれくらいの規模で発生するのかは知っておいても良いのかと思いました。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"focuses on persistent storage options you can configure with your cloud."
+msgstr ""
+"OpenStackのストレージには種類の異なる複数の選択肢があり、経験豊富なクラウドエ"
+"ンジニアでも混乱する場合があります。このセクションでは構成可能な永続的スト"
```
→ストレージは OpenStack スタックの多くの部分に存在し、これらのタイプの違いにより経験豊富なクラウド技術者でさえ混乱する事があります。

s/このセクションでは構成可能な/本章では、あなたのクラウドで設定可能な/

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:29
+msgid "OpenStack Storage"
+msgstr "OpenStackが持つストレージ機能"
```
→OpenStack のストレージ

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:35
+msgid "Block storage"
+msgstr "ブロック・ストレージ"
```
同上。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:41
+msgid "Used to…"
+msgstr "目的"
```
→使用目的

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:44
+msgid "Add additional persistent storage to a virtual machine (VM)"
+msgstr "永続的なストレージをVMへ追加する"
```
→永続的なストレージを仮想マシン（VM）に追加する。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:72
+msgid "Persists until…"
+msgstr "データが残る期間"
```
データの残存期間

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:79
+msgid ""
+"Administrator configures size settings, known as <emphasis>flavors</emphasis>"
+msgstr "管理者が構成した<emphasis>flavors</emphasis>で決まる"
```
→管理者がサイズを設定（<emphasis>flavors</emphasis> として知られる）

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:90
+msgid "10 GB first disk, 30GB second disk"
+msgstr "10GBの初期ディスク、30GBの2台目ディスク"
```
10GB の１台目ディスク、30GB の２台目ディスク

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:94
+msgid "10s of TBs of dataset storage"
+msgstr "数十TBのデータセット"
```
→数十TBのデータセットストレージ

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"your users."
+msgstr ""
+"Novaのみを構成した場合、ユーザはデフォルトで永続ストレージへのアクセス方法を"
+"持ちません。この状態でVMに割り当てられるディスクは”エフェメラル”であり、これ"
```
→デフォルトではユーザにはあらゆる種類の永続ストレージへのアクセス方法がありません。
s/"エフェメラル"/「エフェメラル」/

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+msgstr ""
+"Novaのみを構成した場合、ユーザはデフォルトで永続ストレージへのアクセス方法を"
+"持ちません。この状態でVMに割り当てられるディスクは”エフェメラル”であり、これ"
+"は仮想マシンが終了した時に削除されてしまいます。そのためユーザに対してどんな"
```
→仮想マシンが削除された時にディスクが削除される事を意味します。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:110
+#: doc/src/docbkx/openstack-ops/src/ch_ops_backup_recovery.xml:151
+msgid "Object Storage"
+msgstr "オブジェクト・ストレージ"
```
s/オブジェクト・ストレージ/オブジェクトストレージ/
以下同様

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+#: doc/src/docbkx/openstack-ops/src/ch_ops_backup_recovery.xml:141
+#: doc/src/docbkx/openstack-ops/src/ch_ops_user_facing.xml:410
+msgid "Block Storage"
+msgstr "ブロック・ストレージ"
```
s/ブロック・ストレージ/ブロックストレージ/
以下同様

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"which supports multiple back-ends in the form of drivers. Your choice of a "
+"storage back-end must be supported by a Block Storage driver."
+msgstr ""
+"このボリュームは永続的で、データを残したままデタッチし、別の仮想マシンへ再ア"
```
→これらのボリュームは永続的です。データを残したまま、ボリュームをある仮想マシンからデタッチし、別の仮想マシンに再アタッチする事ができます。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+msgstr ""
+"このボリュームは永続的で、データを残したままデタッチし、別の仮想マシンへ再ア"
+"タッチすることができます。ブロック・ストレージはOpenStack Block Storage "
+"(Cinder) で実装され、複数のバックエンド・ストレージをドライバという形式でサ"
```
→OpenStack では、ブロックストレージは OpenStack Block Storage （Cinder）で実装されており、

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"このボリュームは永続的で、データを残したままデタッチし、別の仮想マシンへ再ア"
+"タッチすることができます。ブロック・ストレージはOpenStack Block Storage "
+"(Cinder) で実装され、複数のバックエンド・ストレージをドライバという形式でサ"
+"ポートします。選択するストレージ・バックエンドはドライバでサポートされている"
```
→あなたが選択するストレージバックエンドは、Block Storage のドライバによりサポートされている必要があります。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"read/write IO."
+msgstr ""
+"多くのストレージ・ドライバはインスタンスが直接ストレージハードウェアへアクセ"
+"スできるようにします。これは read/write IO 性能の向上に役立ちます。"
```
→これは、リード／ライト I/O 性能の向上に役立ちます。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"well as a GlusterFS driver."
+msgstr ""
+"Folsomでは実験的なサポートとしてファイルをボリュームとして利用できます。初期"
+"のCinderでは、NFSを利用するためのリファレンスとしてこのドライバーは実装されま"
```
これは、最初は Cinder で NFS を使用するためのリファレンスドライバとしてスタートしたものです。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+msgstr ""
+"Folsomでは実験的なサポートとしてファイルをボリュームとして利用できます。初期"
+"のCinderでは、NFSを利用するためのリファレンスとしてこのドライバーは実装されま"
+"した。Grizzlyリリースでは、GlusterFSドライバと同様に、完全なNFSドライバとして"
```
→Grizzly リリースまでに、このドライバは GlusterFS ドライバと同様、完全な NFS ドライバに拡張されました。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"machines stored in <code>/var/lib/nova/instances</code>."
+msgstr ""
+"これらのドライバーは従来のブロック・ストレージドライバとは異なる動作をしま"
+"す。NFSやGlusterFSでは1つのファイルが作成され、インスタンスに対して仮想ボ"
```
s/仮想/「仮想」/

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"す。NFSやGlusterFSでは1つのファイルが作成され、インスタンスに対して仮想ボ"
+"リュームとしてマッピングされます。このマッピングはOpenStackで使われる QEMUの"
+"ファイルベースの仮想マシンストア<code>/var/lib/nova/instances</code> に似てい"
+"ます。"
```
→このマッピング／変換は、 <code>/var/lib/nova/instances</code> 下に保存される QEMU のファイルベースの仮想マシンの、OpenStack による扱い方と同様です。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:153
+msgid "File-level Storage"
+msgstr "ファイルレベル・ストレージ"
```
s/ファイルレベル・ストレージ/ファイルレベルストレージ/
以下同様

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"the most common form is called CIFS (previously, SMB)."
+msgstr ""
+"ファイルレベル・ストレージでは、ユーザはOSのファイルシステムインターフェース"
+"を使ってデータへアクセスします。UnixではNFSが一般的で、WindowsではCIFS(SMB)が"
```
欠けた一文です。
→ほとんどのユーザは（以前ネットワークソリューションを使用した経験があった場合）この種類のネットワークストレージに遭遇したことがあります。

yosshy replied:
s/(SMB）/(旧 SMB）

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"<code>/var/lib/nova/instances</code> when designing your cloud, since you "
+"must have a shared file system if you wish to support live migration."
+msgstr ""
+"OpenStackではエンドユーザがファイルレベル・ストレージの恩恵を直接受けることは"
```
s/の恩恵を直接受ける/を目にする/

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"ありません。しかし管理者のクラウド設計においてライブマイグレーションをサポー"
+"トしたい場合、インスタンスデータの保存先 <code>/var/lib/nova/instances</"
+"code> にファイルレベル・ストレージによる共有ファイルシステムの導入を検討する"
+"必要があります。 "
```
→しかし、クラウドの設計時、<code>/var/lib/nova/instances</code> 下のインスタンスの保存用にファイルレベルストレージを検討する事は重要です。なぜなら、ライブマイグレーションをサポートしたい場合、共有ファイルシステムが必須だからです。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+msgid ""
+"What is the platter count I can achieve? Do more spindles result in better I/"
+"O despite network access?"
+msgstr "必要な容量とアクセス速度は？"
```
→実現可能な容量は？ネットワークアクセスでも、より多くのディスクがより良い I/O 性能に繋がるか？

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:197
+msgid "Which one results in the best cost-performance scenario I'm aiming for?"
+msgstr "何をターゲットとしてコストパフォーマンスを最適化するか？"
```
→どちらが自分の意図した最高のコストパフォーマンスシナリオを実現するか？
意味としては大して変わらないのですが。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"fails? To what extent can it mitigate my data-loss disaster scenarios?"
+msgstr ""
+"ストレージの冗長性と分散をどうするか？ストレージノード障害でどうなるか？どの"
+"程度までデータ消失を軽減するか？"
```
→災害時、自分のデータ消失をどの程度軽減できるのか？

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"number of open-source packages, as shown in the following table:"
+msgstr ""
+"コモディティ・ハードウェアを利用したストレージ環境を構築するには、以下のオー"
+"プンソースパッケージを利用可能です。"
```
"a number of" が抜けています。
→コモディティハードウェアを使用したストレージ環境の構築に、下記の表に示したいくつかのオープンソースパッケージが利用可能です。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+msgstr ""
+"* このOSS ファイルレベル・共有ストレージのリストは完全ではなく、他にもOSSが存"
+"在します(MooseFS)。あなたの組織では既に利用可能なファイルレベル・共有ストレー"
+"ジが利用できる可能性もあります。"
```
s/利用できる可能性もあります/あるかも知れません/

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+msgstr ""
+"クラウド内でオブジェクト・ストレージの利用を検討する必要があります。コン"
+"ピュート・クラウドでの一般的なオブジェクト・ストレージ利用方法は以下の二つで"
+"す。"
```
→コンピュートクラウドで提供されるオブジェクトストレージの一般的な利用方法は

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"different commodity storage back-end technologies."
+msgstr ""
+"このセクションでは様々な コモディティ・ハードウェアを利用するストレージ技術の"
+"差異について、概要を提供します。"
```
→上位レベルの概要を

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"multiple data center deployment through support of asynchronous eventual "
+"consistency replication."
+msgstr ""
+"<emphasis role=\"bold\">OpenStack Object Storage (Swift)</emphasis> OpenStack"
```
→emphasis>。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"Storage. More detail can be found about OpenStack Object Storage in the "
+"section below."
+msgstr ""
+"複数データセンターにまたがった分散ストレージ・クラスタを計画する場合や、コン"
```
"Therefore, ～eventually" が抜けています。
→従って、将来的に

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"section below."
+msgstr ""
+"複数データセンターにまたがった分散ストレージ・クラスタを計画する場合や、コン"
+"ピュートとオブジェクト・ストレージ間で統一されたアカウントを必要とする時、ま"
```
前後が「～場合」なので、ここも s/時/場合/ とした方が良いでしょう。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+msgstr ""
+"複数データセンターにまたがった分散ストレージ・クラスタを計画する場合や、コン"
+"ピュートとオブジェクト・ストレージ間で統一されたアカウントを必要とする時、ま"
+"たは OpenStack dashboard を使ってオブジェクト・ストレージを操作したい場合など"
```
原文がそうですが、固有名詞なので s/dashboard/Dashboard/ の方が良いでしょう。

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"複数データセンターにまたがった分散ストレージ・クラスタを計画する場合や、コン"
+"ピュートとオブジェクト・ストレージ間で統一されたアカウントを必要とする時、ま"
+"たは OpenStack dashboard を使ってオブジェクト・ストレージを操作したい場合など"
+"に OpenStack Object Storage を検討します。より詳細な情報は以後のセクションで"
```
→OpenStack Object Storage のより詳細な…

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"developed by one of the founders of DreamHost and is currently used in "
+"production there."
+msgstr ""
+"<emphasis role=\"bold\">Ceph</emphasis> ストレージ・ノード間でデータレプリ"
```
→/emphasis>。
→コモディティなストレージノード間で

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:74
+#: doc/src/docbkx/openstack-ops/src/ch_arch_storage.xml:75
+msgid "Deleted by user"
+msgstr "ユーザが削除するまで"
```
s/ユーザ/ユーザー/
以下同様

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"計されています： オブジェクト・ストレージ、ブロック・ストレージ、ファイルシス"
+"テムをサポートしていますが、ファイルシステムはまだプロダクションレディではあ"
+"りません。Cephはオブジェクト・ストレージでSwiftと同じAPIをサポートし、Cinder "
+"ブロックストレージのバクエンドとしても利用でき、Glance用イメージのバックエン"
```
s/バクエンド/バックエンド/

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+"images. Ceph supports \"thin provisioning\", implemented using copy-on-write."
+msgstr ""
+"Ceph はエンドユーザに対して異なるストレージインターフェースが利用できるよう設"
+"計されています： オブジェクト・ストレージ、ブロック・ストレージ、ファイルシス"
```
→設計されています。Ceph はオブジェクトストレージ、…

yosshy commented on doc/src/docbkx/openstack-ops/locale/chap/ch_arch_storage.xml.ja.po:
```diff
@@ -0,0 +1,877 @@
+msgstr ""
+"Ceph はエンドユーザに対して異なるストレージインターフェースが利用できるよう設"
+"計されています： オブジェクト・ストレージ、ブロック・ストレージ、ファイルシス"
+"テムをサポートしていますが、ファイルシステムはまだプロダクションレディではあ"
```
s/プロダクションレディ/商利用可能な状態/

Commit:
s/ブロック・ストレージ/ブロックストレージ/ at L.244

This message string is shared with other files and it needs to
be synced to avoid fuzzy during merge.

irixjp committed:
fixed ch_arch_storage.xml

irixjp committed:
delete separator

amotoki committed:
Remove unnecessary newline(s)
